build/XCBuildData/05b41b3c725f9c5616c0927687376afa.xcbuilddata/build-request.json

```diff
@@ -0,0 +1,27 @@
+{
+  "buildCommand" : {
+    "command" : "build",
+    "skipDependencies" : false,
+    "style" : "buildOnly"
+  },
+  "configuredTargets" : [
+
+  ],
+  "continueBuildingAfterErrors" : false,
+  "dependencyScope" : "workspace",
+  "enableIndexBuildArena" : false,
+  "hideShellScriptEnvironment" : false,
+  "parameters" : {
+    "action" : "build",
+    "overrides" : {
+
+    }
+  },
+  "qos" : "default",
+  "schemeCommand" : "launch",
+  "showNonLoggedProgress" : true,
+  "useDryRun" : false,
+  "useImplicitDependencies" : false,
+  "useLegacyBuildLocations" : false,
+  "useParallelTargets" : true
+}
```

Comment:
Not sure what these are, I'm guessing you did a git add . and some things snuck through that shouldn't have?

Reply:
Looks like these are all build data files. They shouldn't be in a commit like this - try go to Product -> Clean build folder -> and then do a git add again. Hopefully they should disappear then.